packages/modal/src/Modal.spec.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { render, cleanup } from '@testing-library/react';
+import 'jest-dom/extend-expect';
+import Modal from './Modal';
+
+afterAll(cleanup);
+
+describe('packages/Modal', () => {
+  const modalId = 'modal';
+  const modalContent = 'Modal Content';
+
+  describe('when rendered with content', () => {
+    const { getByText, getByTestId } = render(
+      <Modal active data-testid={modalId}>
+        <h4>{modalContent}</h4>
+      </Modal>,
+    );
+
+    test('it appears on page when active prop is set', () => {
+      const modal = getByTestId(modalId);
+      expect(modal).toBeVisible();
+    });
+
+    test('it is rendered to the DOM when the active prop is set', () => {
+      const modal = getByTestId(modalId);
+      expect(modal).toBeInTheDocument();
+    });
+
+    test(`it renders the ${modalContent} as expected`, () => {
```

Comment:
Kinda weird to interpolate the actual value. we just want to make sure it renders our content, not what the content is necessarily. That's up to the assertion.

Reply:
👍